pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentName.java

```diff
@@ -73,4 +74,30 @@ public int getPartitionId() {
   public String getPartitionRange() {
     throw new RuntimeException("No partitionRange in " + getSegmentName());
   }
+
+  public static boolean isHighLevelConsumerSegmentName(String segmentName) {
+    if (segmentName.endsWith(SEPARATOR) || segmentName.startsWith(SEPARATOR)) {
+      return false;
+    }
+
+    String[] parts =  segmentName.split(SEPARATOR);
+    if (parts.length != 3 && parts.length != 5) {
```

Comment:
It's already documented in other parts (see https://github.com/linkedin/pinot/pull/252/files#diff-5aae5dfc010ab141a3c61f94cdebc42aR136 and https://github.com/linkedin/pinot/pull/252/files#diff-ae33b4a0fa4da0354102626cfcc98a0cR37) and most of this code should go away when we remove the high level consumer.